utils/roles/oim_cleanup/oim_container_cleanup/vars/main.yml

```diff
@@ -35,6 +35,7 @@ pulp_cleanup_directory:
   - "{{ omnia_nfs_share }}/pulp/pulp_ha/cli.toml"
   - "{{ omnia_nfs_share }}/log/pulp"
   - "{{ omnia_nfs_share }}/pulp/settings"
+offline_repo_path_dir: "/opt/omnia/offline_repo/"
```

Comment:
can we append the path in pulp_cleanup_directory instead of creating new variable